wiki/osu!_wiki/Contribution_guide/Best_practices/en.md

```diff
@@ -68,6 +68,18 @@ A pull request shows other people how your edits will affect the files. Add some
   - [Automatic resolution of relevant issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue)
 - Make sure to tick the `Allow edits from maintainers` checkbox, as it will allow the wiki maintainers to help you improve the pull request when necessary
 
+## Applying reviews
+
+Reviews are best applied directly through the github web interface. Use the `Add suggestion to batch` button when in the `Files changed` tab to apply multiple reviews simultaneusly.
+
+![][img/apply-batch-review.gif]
+
+You may also use the `Commit suggestion` button to apply a single suggestion individually, but don't forget to set a descriptive commit message. Keep in mind however, that fewer related commits are better than lots of small ones.
+
+![][img/apply-single-review.gif]
+
+Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didnt add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Applying reviews automatically is preferred, as it prevents errors and alleviates reviewers from potentially needing to check if their review was applied correctly, especially when there are many changes.
```

Comment:
```suggestion
Using this system will automatically mark suggestions as resolved. When applying reviews manually (e.g. when the reviewer didn't add a direct suggestion), mark them as resolved *after committing the change* to prevent forgetting any. Applying reviews automatically is preferred, as it prevents errors and alleviates reviewers from potentially needing to check if their review was applied correctly, especially when there are many changes.
```